provider/resources.go

```diff
@@ -5876,3 +5877,7 @@ func setupComputedIDs(prov *tfbridge.ProviderInfo) {
 		return attrWithSeparator(state, ":", "name", "restoreTestingPlanName"), nil
 	}
 }
+
+func init() {
+	log.SetOutput(io.Discard)
```

Comment:
Can we test in any way that the bridge actually resets this again in the end? I'm a bit worried that the bridge behavior will be eventually changed and then this here just send everything into the void for aws

Reply:
Can possibly try testing something like this statement in ACM certificate read:

		log.Printf("[WARN] ACM Certificate %s not found, removing from state", arn)

Reply:
I couldn't find an existing test for this so constructed one using aws.s3.Bucket and object.

Reply:
This is very similar to https://github.com/pulumi/pulumi-databricks/issues/474#issuecomment-2158908967. If this works, then I don't think it needs to be in `init` (I think `init` occurs after all dependencies have been loaded, ordering of Go's life before main is largely UB). If you put this at the top of the `Provider` function, does it not work? That would give you guaranteed scoping with a `defer log.SetOutput(os.Stdout)`.